server.py

```diff
@@ -78,6 +78,21 @@ def get_topic(topic_id):
   resp_data = json.loads(resp.get_data().decode())
   return resp_data
 
+def image_adder(key, item):
+  if key in item and item[key]:
+      for obj in item[key]:
+          try:
+              if 'image' in obj and obj['image']:
+                  query = str(obj['image'])
+                  headers = dict(request.headers)
+                  tc = app.test_client()
+                  resp = tc.get('images?where={"_id":{"$in":['+ '"' + query + '"' + ']}}', headers=headers)
```

Comment:
Since only one image per author, you can get image by `images/${query}`. There is no need to use `where` parameter.